com.unity.robotics.urdf-importer/Runtime/RosSharpDefinitions/UrdfJointDescription.cs

```diff
@@ -95,45 +96,66 @@ public void WriteToUrdf(XmlWriter writer)
 
         public class Axis
         {
-            public double[] xyz;
 
-            public static Axis DefaultAxis => new Axis(new double[] {1, 0, 0});
+            public Vector3 axisROS;
+            
+            const string _AtrributeName = "xyz";
 
-            public Axis(XElement node)
+            private static Vector3 GetDefaultAxisValues()
             {
-                xyz = node.Attribute("xyz") != null ? node.Attribute("xyz").ReadDoubleArray() : null;
+                return new Vector3(1, 0, 0);
```

Comment:
Should we just use Vector.right here? It's perhaps a little harder to read?

Reply:
Added a comment. I think it should be kept as new Vector3(1, 0, 0);